corehq/apps/app_manager/static_strings.py

```diff
@@ -160,6 +160,8 @@
     gettext_noop('Server User Registration'),
     gettext_noop('Set to skip if your deployment does not require users to register with the server. Note that this will likely result in OTA Restore and other features being unavailable.'),
     gettext_noop('Short'),
+    gettext_noop('Show a persistent menu instead of the bread crumbs bar. This feature is work in progress.'),
```

Comment:
```suggestion
    gettext_noop('Show a persistent menu instead of breadcrumbs.'),
```

Reply:
We are generally bad about updating text like this ("in progress"). I'm curious, why isn't this a feature flag?

Reply:
@orangejenny So you don't think I should put any warning on there that this does not do anything yet? This will be visible with the next deploy.

Reply:
Why is this being released if it doesn't do anything? I think you should flag it or otherwise hide it.

Reply:
Cody did not think it needed a FF. But I guess he was thinking about when the whole thing is done. Will add a FF.